features/step_definitions/admin_auction_form_steps.rb

```diff
@@ -35,6 +35,51 @@
   select("reverse", from: "auction_type")
 end
 
+When(/^I fill auction form with choosing "([^"]*)" purchase card$/) do |purchase_card|
```

Comment:
since we are only using this step for the `other` purchase card, and it does something that only the `other ` purchase card can do (publish on `create`) -- how about hard-coding the step name instead of using interpolation? Something like `"I create and publish an auction for the non-default purchase card"`